.gitignore

```diff
@@ -6,3 +6,4 @@ VERSION
 *.rbc
 *.swp
 /.bundle
+.idea/
```

Comment:
IntelliJ stores details in as `.idea/*` directory, it's not a file like `.bundle`.